pkg/kubelet/server.go

```diff
@@ -346,6 +352,18 @@ func (s *Server) InstallDebuggingHandlers() {
 	s.restfulCont.Add(ws)
 }
 
+// InstallApiHandlers registers the HTTP request patterns that serve the kubelet versioned api.
+func (s *Server) InstallApiHandlers() {
+	ws := new(restful.WebService).
+		Path("/api/v1/").
+		Consumes(restful.MIME_JSON).
+		Produces(restful.MIME_JSON)
+
+	ws.Route(ws.GET("rawMetrics").To(handlers.RawMetricsHandler(s.host, s.host.GetStatusManager())))
```

Comment:
Some background: We want to switch to using the apiserver library in the kubelet as well. This is currently not possible, but it has been a want for a while now. 
To make that transition possible, we should try to stick to the general k8s API conventions even at the kubelet level and match the main K8s APIs if applicable.
K8s API semantics requires all resources to support `Get` operation. 

What would `metrics` and `metrics/[raw|derived` return based on your proposal @jimmidyson? I guess the sub-resources?
We do not have a resource (e.x. `metrics`) that is meant for just grouping in the core API. 
@bgrant0607 suggested using api-groups in the future for this purpose.

Until we can switch to using api-groups at the kubelet level, I'd prefer re-using the terminology and schema we have agreed upon for the Kubernetes API, which is `derivedMetrics` (and `rawMetrics` if we introduce it in the future).

PS: IIUC, with api-groups, we should be able to define a `apis/metrics` group, and then have `raw` and `derived` as resources.